pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -17,7 +17,7 @@ callPackage ./generic.nix args {
   # check the release notes for compatible kernels
   kernelCompatible = kernel.kernelOlder "6.11";
 
-  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_9;
+  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_6;
```

Comment:
It says `kernelOlder "6.11"` above, so we should be able to use linux_6_10 here?

Reply:
I think that might be wrong, because I checked the sources, and it said 6.8.

Reply:
Below are apparently a few 6.10 compat patches in `unstable.nix` that were approved by upstream.

Reply:
oh right! thanks